src/main/resources/public/jsx/geodashreact.js

```diff
@@ -333,25 +360,31 @@ class MapWidget extends React.Component {
         if(widget.filterType != null && widget.filterType.length > 0){
             const fts = {"LANDSAT5": "Landsat5Filtered", "LANDSAT7": "Landsat7Filtered", "LANDSAT8":"Landsat8Filtered", "Sentinel2": "FilteredSentinel"};
             url = window.location.protocol + "//" + window.location.hostname + ":8888/" + fts[widget.filterType];
+            url = "https://ceodev.servirglobal.net:8888/" + fts[widget.filterType];
```

Comment:
What is going on with changing url twice?  This function looks like you could just return the url instead of creating a variable